content/en/docs/tasks/observability/distributed-tracing/zipkin/multicluster_test.sh

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env bash
```

Comment:
This test is just a temporary dummy to test the multicluster profile, right? It has nothing to do with the zipkin task, so we should remove it as soon as we have a real multicluster test.

Reply:
yup .. that's the plan.